packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CameraPlugin.m

```diff
@@ -131,8 +134,36 @@ - (void)availableCamerasWithCompletion:
           lensFacing = FCPPlatformCameraLensDirectionExternal;
           break;
       }
+
+      if ([device.deviceType isEqualToString:AVCaptureDeviceTypeBuiltInWideAngleCamera]) {
+        lensType = FCPPlatformCameraLensTypeWide;
+      } else if ([device.deviceType isEqualToString:AVCaptureDeviceTypeBuiltInTelephotoCamera]) {
+        lensType = FCPPlatformCameraLensTypeTelephoto;
+      } else if (@available(iOS 13.0, *)) {
+        if ([device.deviceType isEqualToString:AVCaptureDeviceTypeBuiltInUltraWideCamera]) {
+          lensType = FCPPlatformCameraLensTypeUltraWide;
+        } else if ([device.deviceType isEqualToString:AVCaptureDeviceTypeBuiltInDualCamera]) {
+          lensType = FCPPlatformCameraLensTypeDual;
+        } else if ([device.deviceType isEqualToString:AVCaptureDeviceTypeBuiltInDualWideCamera]) {
+          lensType = FCPPlatformCameraLensTypeDualWide;
+        } else if ([device.deviceType isEqualToString:AVCaptureDeviceTypeBuiltInTripleCamera]) {
+          lensType = FCPPlatformCameraLensTypeTriple;
+        } else {
+          lensType = FCPPlatformCameraLensTypeUnknown;
+        }
+      } else if (@available(iOS 17.0, *)) {
+        if ([device.deviceType isEqualToString:AVCaptureDeviceTypeContinuityCamera]) {
+          lensType = FCPPlatformCameraLensTypeContinuity;
+        } else {
+          lensType = FCPPlatformCameraLensTypeUnknown;
+        }
+      } else {
+        lensType = FCPPlatformCameraLensTypeUnknown;
+      }
+
       [reply addObject:[FCPPlatformCameraDescription makeWithName:device.uniqueID
-                                                    lensDirection:lensFacing]];
+                                                    lensDirection:lensFacing
+                                                         lensType:lensType]];
```

Comment:
Nit: I know you are following the prior convention of `FCPPlatformCameraLensDirection` here, but can you move both `FCPPlatformCameraLensDirection` and `FCPPlatformCameraLensType` to a helper function since it's getting long?

Reply:
Hey @hellohuanlin I made this change